composer.json

```diff
@@ -23,7 +23,7 @@
         "keboola/billing-api-php-client": "^0.1",
         "keboola/configuration-variables-resolver": "^1.0.0",
         "keboola/dockerbundle": "dev-odin-library-3@dev",
-        "keboola/input-mapping": "13.2.0",
+        "keboola/input-mapping": "13.2.2",
```

Comment:
should be able to release the hard version?  like ^13.2.x?

Reply:
OFC! wrong version pushed :facepalm:

Reply:
fixed, thanks